lib/evss/service.rb

```diff
@@ -32,8 +32,8 @@ def handle_error(error)
         log_message_to_sentry(error.message, :error, extra_context: { url: config.base_path })
         raise_backend_exception('EVSS502', self.class)
       when Common::Client::Errors::ClientError
-        raise Common::Exceptions::Forbidden if error.status == 403
         log_message_to_sentry(error.message, :error, extra_context: { url: config.base_path, body: error.body })
+        raise Common::Exceptions::Forbidden if error.status == 403
```

Comment:
@lihanli This is where the `Forbidden` error originates.  I moved it below the call to `log_message_to_sentry` to ensure the original error gets logged.  Do you think this suffices for logging?

Reply:
that should work, does that look ok @kreek?

Reply:
👍 works for me